CHANGELOG.md

```diff
@@ -23,3 +23,13 @@
   * `UNOCONV_SERVER` and `UNOCONV_PORT` are no longer supported configuration options. Please remove
 	them from your configuration file.
   * By default an unoconv process gets launched within the container.
+
```

Comment:
Please put the changelog for the newer version on top of the file

Reply:
Done. How did I miss this :-)